third_party/libusb/naclport/src/libusb_over_chrome_usb.cc

```diff
@@ -523,6 +533,17 @@ int LibusbOverChromeUsb::LibusbOpen(
   if (!result.is_successful()) {
     GOOGLE_SMART_CARD_LOG_WARNING << "LibusbOverChromeUsb::LibusbOpen " <<
         "request failed: " << result.error_message();
+    // modify the devices (fake) bus number that we report so that PCSC will 
+    // retry to connect to the device once it updates the device list
+    if (kBusNumbers.find(dev->chrome_usb_device().device) != 
+        kBusNumbers.end()) {
+      kBusNumbers[dev->chrome_usb_device().device] = std::min(
+          static_cast<uint8_t>(
+          kBusNumbers[dev->chrome_usb_device().device] + 1),
+          kMaximumReconnectAttempts);
+    } else {
+      kBusNumbers[dev->chrome_usb_device().device] = kDefaultBusNumber + 1;
```

Comment:
nit: Similar to above, let's have a single line that performs the update in the map, and calculate the bus number in advance.

Or, actually, can we use LibusbGetDeviceAddress() here? Then we won't have two different places that hardcode kDefaultBusNumber.

Reply:
Done